test/e2e/config/docker-ci.yaml

```diff
@@ -71,7 +71,7 @@ variables:
 intervals:
   default/wait-controllers: ["3m", "10s"]
   default/wait-cluster: ["3m", "10s"]
-  default/wait-control-plane: ["5m", "10s"]
+  default/wait-control-plane: ["10m", "10s"]
```

Comment:
Did you observed errors related to this timeout? This should not be affected by the changes in this PR...